modules/efk-logging-elasticsearch-storage.adoc

```diff
@@ -25,11 +37,30 @@ metadata:
     logStore:
       type: "elasticsearch"
       elasticsearch:
-        nodeCount: 3
+        nodeCount: 3 
         storage:
           storageClass: 
-            name: "gp2"
-            size: "200G"
+            name: "gp2" <1>
+            size: "200G" <2>
 ----
+<1> Specify the Storage Class name.
+<2> Specify the size of the request.
+
+.. Edit the  Cluster Logging CR to specify emptyDir:
+
+[source,yaml]
+----
+apiVersion: "logging.openshift.io/v1alpha1"
+kind: "ClusterLogging"
+metadata:
+  name: "clusterlogging"
+
+....
 
-This example specifies each data node in the cluster is bound to a Persistent Volume Claim that requests "200G" of AWS General Purpose SSD (gp2) storage.
+ spec:
+    logStore:
+      type: "elasticsearch"
+      elasticsearch:
+        nodeCount: 3 
+        storage: {}
```

Comment:
@mburke5678 Is this section  for ephemeral storage?

Reply:
@anpingli Sorry. I edited the wrong files. Please see: https://github.com/openshift/openshift-docs/pull/13877/commits/bac6800792bb42df8e80e925832ad5ac39b94ddf